### PR TITLE
add purge_quadlet_dir param

### PR DESCRIPTION
**Pull request by jhoblitt**

### CODE REVIEW

**Comment by jhoblitt** [comment]
I'm guessing F40 is broken everywhere?

File: manifests/config.pp [hunk 1]
```diff
@@ -5,11 +5,14 @@
...
-      ensure => directory,
-      owner  => root,
-      group  => root,
-      mode   => '0644',
+    file { $quadlets::quadlet_dir:
+      ensure  => directory,
+      owner   => root,
+      group   => root,
+      mode    => '0644',
+      purge   => $quadlets::purge_quadlet_dir,
```
Comment (traylenator) [inline manifests/config.pp:13]: Can we be consistent with the existing variables and set them local in this class.

**Review by traylenator — CHANGES_REQUESTED** [review]

File: manifests/config.pp (hunk 1 above)
Comment (jhoblitt) [reply manifests/config.pp:13]: I'm generally not a fan of using inheritance for passing internal data but OK.  Note that `quadlets::quadlet` will still have to directly to the variable in `quadlets`.

**Comment by traylenator** [comment]
Tomorrow.

**Review by traylenator — APPROVED** [review]

**Pull request merged**